### ISSUE TITLE
question about collectionEntity query

### ISSUE THREAD

**Issue by roiLeo** [kodadot/rubick#127]
Is there a way to fetch TotalCount of NFTEntity from a particular collection?

Instead of doing this query that would retrieve all nfts from collection
```graphql
query collectionListByAccount($account: String!, $first: Int, $offset: Int) {
  collectionEntities(
    where: { issuer_eq: $account, nfts_some: { burned_eq: false } }
    orderBy: blockNumber_DESC
    limit: $first
    offset: $offset
  ) {
      name
      id
      nftEntities: nfts(where: {burned_eq: false}) {
        id, 
        name
      } 
    }
}
```

I would like to know if there is a way to link `collectionEntities` with `nftEntitiesConnection` in the same query?

**Comment by vikiival** [comment]
Yup, 

https://github.com/kodadot/rubick/blob/main/src/server-extension/resolvers/count.ts

Just uncomment the `totalTokensByCollectionId` :)

Make PR and happy to merge :) 

also it would be maybe nice to extend it with the clause if they are burned or not

**Comment by roiLeo** [comment]
Wrong project :/
Can't we just make a "LEFT JOIN" with current collectionId ? I'm really dumped with typescrript backend

**Comment by vikiival** [comment]
The other option is to fetch the total amount of NFTs from collection via `nftEntityConnection`

I would provide you an example but 🇪🇸 🚆 has terrible signal.

>  I'm really dumped with typescrript backend

No worries 🤗. Happy to help! 
Also @petersopko || @KngZhi || @Jarsen136 can help

**Comment by KngZhi** [comment]
```graphql
query MyQuery($id: String!) {
  collectionEntitiesConnection(
    where: { id_eq: $id, nfts_some: { burned_eq: false } }
  ) {
    totalCount
  }
}
```
may this help :)

**Comment by roiLeo** [comment]
> ```graphql
> query MyQuery($id: String!) {
>   collectionEntitiesConnection(
>     where: { id_eq: $id, nfts_some: { burned_eq: false } }
>   ) {
>     totalCount
>   }
> }
> ```
> 
> may this help :)

I don't have `$id` to provide :)

see my query in first comment, I wanted to list all collections created by user & have in each collection object a total number of nfts. Instead of retrieve all nfts & doing `nfts.length`

**Comment by KngZhi** [comment]
in Rubick, can't directly do that, we have to write a resolver for that.

**Comment by roiLeo** [comment]
closed by #153
